sys-libs/libcxx/libcxx-9.0.0.ebuild

```diff
@@ -93,7 +93,7 @@ src_configure() {
 multilib_src_configure() {
 	# we want -lgcc_s for unwinder, and for compiler runtime when using
 	# gcc, clang with gcc runtime (or any unknown compiler)
-	local extra_libs=() want_gcc_s=ON
+	local extra_libs=() want_gcc_s=ON want_compiler_rt=NO
```

Comment:
Why not ON/OFF for consistency? Also, could you check if that gcc_s logic is still needed, or does one imply the other?

Reply:
Nice catch! Thanks for pointing it out.

I just checked https://github.com/llvm-mirror/libcxx/blob/master/cmake/config-ix.cmake#L40 , gcc_s is still needed if compiler-rt is not linked.